cloudbuild.yaml

```diff
@@ -1,5 +1,35 @@
 steps:
-- name: 'gradle:5.6-jdk11'
+- name: 'gcr.io/cloud-builders/git'
+  id: clone-deid
+  args:
+  - clone
+  - https://github.com/red1408/healthcare-deid.git
+  - healthcare-deid
+
+- name: 'gcr.io/cloud-builders/git'
+  id: checkout-deid-branch
+  args:
+  - checkout
+  - "stream_edit"
+  dir: healthcare-deid
+
+- name: ${_GRADLE_IMAGE}
+  id: deid-redactor-install
+  volumes:
+  - name: 'vol1'
+    path: '/root/.m2'
+  args:
+  - gradle
+  - install
+  dir: healthcare-deid/offline_tools/redactor
+
+- name: ${_GRADLE_IMAGE}
+  volumes:
+  - name: 'vol1'
+    path: '/root/.m2'
   args:
   - gradle
   - build
+
+substitutions:
+  _GRADLE_IMAGE: 'gradle:5.6-jdk11'
```

Comment:
We shouldn't be forcing users to clone this repo in order to build the adapter.
While we wait for the maven package, is it possible for us to use jitpack.io to package the dependency here?

Reply:
Working on it, couldn't get it to work right away.

Reply:
Ok, feel free to post any issues you may have here.

Reply:
What were the issues you ran into with jitpack.io? Anything we can do to unblock this?

Reply:
It does expect single project in repository, with build file at top level, not collection of projects. It is possible to write custom script in jitpack.yml at top level, but there is no documentation for this (https://jitpack.io/docs/BUILDING/#custom-commands is all there is on topic, and neither looking through first few pages of jitpack issues on github helped). I got it to install the project to local maven, but then jitpack fails to find it ('ERROR: No build artifacts found' after gradle finished build).

I could either try to extract redactor-deid-lib into self-contained project, or maybe go entirely different way, using it as git submodule from adapter.

Reply:
I did some searching and it couldn't find much either. If you can't get the custom build script to work, we might need to wait for a maven repo for the redactor.

Reply:
Extracted redactor to separate repo, using it from jitpack.

Reply:
Update integration test use redactor, this involves few extra changes to work around UID replacement that redactor does by default (not configurable). Also organized integration test files into scripts and data subfolders.